gravitee-apim-rest-api/gravitee-apim-rest-api-management/gravitee-apim-rest-api-management-rest/src/main/java/io/gravitee/rest/api/management/rest/resource/configuration/dictionary/DictionaryResource.java

```diff
@@ -150,7 +150,7 @@ public Response undeployDictionary(@PathParam("dictionary") String dictionary) {
             @ApiResponse(code = 500, message = "Internal server error"),
         }
     )
-    @Permissions({ @Permission(value = RolePermission.APPLICATION_DEFINITION, acls = RolePermissionAction.DELETE) })
+    @Permissions({ @Permission(value = RolePermission.ENVIRONMENT_DICTIONARY, acls = RolePermissionAction.DELETE) })
```

Comment:
I know this test doesn't cover permission, but I'm not sure it's useful to do a specific test for that 🤷‍♂️